src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapterImpl.cs

```diff
@@ -201,29 +201,41 @@ internal int StopScan()
             return ret;
         }
 
+        private string GetPacketTypeToString(BluetoothLePacketType packetType)
+        {
+            return (packetType == BluetoothLePacketType.BluetoothLeAdvertisingPacket) ? "AdvertisingPacket" : "ScanResponsePacket";
+        }
+
         internal IList<string> GetLeScanResultServiceUuids(BluetoothLeScanData scanData, BluetoothLePacketType packetType)
         {
             if (!BluetoothAdapter.IsBluetoothEnabled || !Globals.IsInitialize)
             {
                 BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
             }
 
-            IList<string> list = new List<string>();
             IntPtr uuidListArray = IntPtr.Zero;
             int count = -1;
             BluetoothLeScanDataStruct scanDataStruct = BluetoothUtils.ConvertLeScanDataToStruct(scanData);
 
             int ret = Interop.Bluetooth.GetScanResultServiceUuid(ref scanDataStruct, packetType, ref uuidListArray, ref count);
-            if (ret != (int)BluetoothError.None)
+            if (ret == (int)BluetoothError.NoData)
+            {
+                Log.Info(Globals.LogTag, "No ServiceUuids in " + GetPacketTypeToString(packetType));
```

Comment:
Why do you made a convert method?
It already convert to string like "BluetoothLeAdvertisingPacket" without your convert method

Reply:
I remove the convert method. Thank you.